docs/faq.md

```diff
@@ -302,14 +304,12 @@ The provider for this context is rendered by `BaseForm`, and in turn all the oth
 
 There are two most common issues causing this problem:
 
-1. The component calling this function does not have a Form component above it anywhere in the component tree. To fix
-
-   this, wrap this component within a parent Form component (does not have to be direct).
+1. **The component calling this function does not have a Form component above it anywhere in the component tree.**
 
-2. There are multiple versions of `uniforms` installed in your `node_modules`. This usually happens when you have
+   To fix this, wrap this component within a parent Form component (does not have to be direct).
 
-   more than one version of the core `uniforms` package installed which can happen when you have a mismatch of
+2. **There are multiple versions of `uniforms` installed in your `node_modules`**.
 
-   versions between any of your `uniforms` related dependencies. Ensure all your versions are matching,
+   This usually happens when you have installed more than one version of the core `uniforms` package. It can happen when you have a mismatch of versions between any of your `uniforms` related dependencies.
 
-   clean any `node_modules` directories and reinstall dependencies to resolve this error.
+   Ensure all your versions match, clean any `node_modules` directories and reinstall dependencies to resolve this error.
```

Comment:
Maybe let's put it straight forward and say that by 'match', we mean 'they are the same'.